devel/libuv/Portfile

```diff
@@ -56,3 +56,33 @@ depends_build       port:automake \
                     port:autoconf \
                     port:libtool \
                     port:pkgconfig
+
+
+
+
+
+platform darwin 8 {
+
+    # pegged version with patches for Tiger, updated occasionally
+
+    github.setup    libuv libuv 1.23.2 v
+    revision        1
+    checksums       rmd160 7fda655697a845693008daa9e0a483d15d6214d8 \
+                    sha256 f5c360b53c5d5c22c4429af9e8cb61c9504d176a4bdd533813aaead78447730c \
+                    size   1192667
+
+    maintainers-prepend {kencu @kencu}
+    long_description  ${long_description} This version is pegged for Tiger and is updated occasionally. \
+                      Improvements are welcome if you can improve the test suite success (a few tests fail).
+    configure.cppflags-append -D__DARWIN_UNIX03
+    # prevent conflicting opentransport header from being pulled in
+    configure.cppflags-append -D__OPENTRANSPORTPROVIDERS__
+
+    # delete any patchfiles that may be added above later
+    patchfiles
+    # Tiger unified patch
+    patchfiles-append   patch-libuv-1-23-2-tiger.diff
```

Comment:
Please combine the prior "patchfiles" & this one by just using "patchfiles" here.

Reply:
yes, that accomplishes the same end, more neatly. Thanks :>